tools/automation/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "built-in-extensions-update-automation",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Scripts for updating, adding, removing, and building extensions in che-theia",
+  "main": "check-extension-updates.js",
+  "scripts": {
+    "build": "yarn run format && yarn run compile && yarn run lint",
+    "compile": "tsc --project .",
+    "format": "prettier --write **/*.ts **/**/*.ts package.json",
+    "format-check": "prettier --check **/*.ts **/**/*.ts package.json",
+    "lint": "eslint . --ext .ts",
```

Comment:
linter seems to be failing

Reply:
I am having trouble with this one:

```bash
$ eslint --cache=true --no-error-on-unmatched-pattern=true "{src,test}/**/*.{ts,tsx}"

/home/ericwill/git/che-theia/tools/automation/src/check-extension-updates.ts
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: tools/automation/src/check-extension-updates.ts.
The file must be included in at least one of the projects provided
```

but I included it in the root `tsconfig.json`:
```js
"include": [
        "dev-packages/*/src",
        "plugins/*/src",
        "extensions/*/src",
        "tools/automation/*/src"
    ],
```

what am I missing?